tmail-backend/integration-tests/jmap/jmap-integration-tests-common/src/main/scala/com/linagora/tmail/james/common/LinagoraEchoMethodContract.scala

```diff
@@ -12,6 +13,11 @@ import org.apache.james.jmap.rfc8621.contract.Fixture.{ACCEPT_RFC8621_VERSION_HE
 import org.apache.james.utils.DataProbeImpl
 import org.assertj.core.api.Assertions.assertThat
 import org.junit.jupiter.api.{BeforeEach, Test}
+import org.mockito.Mockito.mock
+
+object LinagoraEchoMethodContract {
+  val firebasePushClient: FirebasePushClient = mock(classOf[FirebasePushClient])
```

Comment:
Why for the echo method contract?

Reply:
We had a firebase capability object test in there before => enable firebase => we need to mock the firebase client for it, otherwise, it fails.